docs/developer-docs/functionality/http_requests/http_requests-how-it-works.md

```diff
@@ -0,0 +1,152 @@
+# Technology Overview -- How Canister HTTP Requests Work
+
+We next give some more details on how canister HTTP requests work and important aspects to consider when using the API. We already want to note here that there are some limitations compared to regular (Web 2.0) computer programs making HTTP outcalls and considerations for programmers for successfully using this feature. Engineers who intend to use this feature are advised to read through this documentation to get up to speed quickly w.r.t. the feature.
```

Comment:
`We next give...` is a weird way to start a page.

`read through this documentation` feels weird. Maybe 'the entire section of the documentation' or 'this page' flows better?

Reply:
Resolved.